docs/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "wikit-docs",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Storybook for illustrating design tokens and components",
+  "keywords": [
+    "wikit",
+    "docs",
+    "storybook",
+    "tokens",
+    "components"
+  ],
+  "author": "The Wikidata team",
+  "homepage": "https://github.com/wmde/wikit/tree/master/docs#readme",
+  "license": "BSD-3-Clause",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wmde/wikit.git"
+  },
+  "scripts": {
+    "storybook": "start-storybook"
+  },
+  "dependencies": {
+    "@wmde/wikit-tokens": "^0.0.0",
+    "@wmde/wikit-vue-components": "^0.0.0",
+    "vue": "^2.6.11"
+  },
+  "bugs": {
+    "url": "https://github.com/wmde/wikit/issues"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.10.3",
+    "@storybook/addon-docs": "^5.3.19",
+    "@storybook/vue": "^5.3.19",
+    "babel-loader": "^8.1.0",
+    "babel-preset-vue": "^2.0.2",
+    "react": "^16.13.1",
+    "react-is": "^16.13.1",
+    "vue-loader": "^16.0.0-beta.4",
```

Comment:
Not sure if the package has this particular version for a reason, but it's spamming the logs when I run storybook:
```
storybook_1  | wikit-docs: WARN   Failed to load preset: "/app/docs/node_modules/@storybook/vue/dist/server/framework-preset-vue.js"
storybook_1  | wikit-docs: ERR! Error: Cannot find module 'vue-loader/lib/plugin'
...
```
Removing it and installing whatever version npm likes best worked for me:
1. `docker-compose run --rm node lerna exec 'npm uninstall --save-dev vue-loader' --scope wikit-docs`
2. `docker-compose run --rm node lerna add vue-loader --dev --scope wikit-docs`

This installed vue-loader 15.9.3 for me, and it didn't break anything as far as I can tell.

Reply:
I guess that happened because I installed storybook manually. 
Thanks!